src/components/WebexActivity/WebexActivity.stories.js

```diff
@@ -2,56 +2,84 @@ import React from 'react';
 import {storiesOf} from '@storybook/react';
 import {addDays, getDay, subDays} from 'date-fns';
 
-import {ActivitiesJSONAdapter, PeopleJSONAdapter} from '../../adapters';
-import {activities, people} from '../../data';
-
-import WebexActivity from './WebexActivity';
+import * as webexData from '../../data';
```

Comment:
If you make it a default export you won't need to use `*`.
Also, can you rename this to `testData` or `jsonData` as to not confuse it with the adapter provider?